ros_buildfarm/status_page.py

```diff
@@ -873,6 +928,98 @@ def _get_blocked_releases_info(config_url, rosdistro_name, repo_names=None):
     return repos_info
 
 
+def _has_source_entry(repo_name, dist_file):
```

Comment:
Since this function is only used in a single place I would suggest to just place the condition in that location. (As an alternative at least move this function directly below the function it is used in.)

*Update:* I noticed that it follows the existing function `_is_released`. With the current patch that function is only called in a single place (`_released_repos`). Therefore I suggest the same for that function.

Reply:
I moved `_has_source_entry()` and `_is_released()` below where they're used in 9109dc09da9f3821955c2f1a65203463729be16f and 2ec5472787fbe4da42879693346f7633daeea0f9. I chose that over inlining because the function name was helpful to me when trying to understand the logic.